nx-dev/nx-dev/redirect-rules.js

```diff
@@ -656,7 +656,7 @@ const packagesIndexes = {
   '/nest': '/packages/nest',
   '/next': '/packages/next',
   '/node': '/packages/node',
-  '/react': '/packages/react',
+  // '/react': '/packages/react',
```

Comment:
Should we have redirects for this?

Reply:
I talked with @isaacplmann about it and said that the redirects were from older pages, but since we're planning on using this route again, we will need to remove it